lib/variations.js

```diff
@@ -13,7 +13,7 @@ const variations = {
     Advise = new Advisor();
     Advise.start();
   },
-  'observe-only'() {},
+  // 'observe-only'() {},
```

Comment:
disabling the 'observe-only' branch of the study for development. It's been annoying to use `jpm run